src/js/gi/components/profile/HeadingSummary.jsx

```diff
@@ -12,6 +12,7 @@ class HeadingSummary extends React.Component {
 
   render() {
     const it = this.props.institution;
+    it.type = it.type && it.type.toLowerCase();
```

Comment:
The data we get can be a bit patchy and have `null` for various values. In general, I doubt we'll have an issue of a `null` type, but this is just to ensure there's no error here.